File: gems/fact-mine/tests/fixtures/stdlib_registry.java
import java.util.Arrays;
import java.util.Objects;

class Registry {
  int inspect(int[] values, String text) {
    Objects.requireNonNull(text);
    int[] copy = Arrays.copyOf(values, values.length);
    return Arrays.binarySearch(copy, 1) + (int) Math.sqrt(copy.length);
  }
}

File: gems/fact-mine/tests/fixtures/stdlib_registry.go
package registry

import (
  "maps"
  "slices"
  "strings"
)

func inspect(text string, values []int, labels map[string]int) bool {
  _, _ = slices.BinarySearch(values, 1)
  _ = maps.Clone(labels)
  return strings.HasPrefix(text, "x")
}
